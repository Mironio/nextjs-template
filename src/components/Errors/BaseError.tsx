import Image from 'next/image';
import exclamation from '/public/images/exclamation.svg';
import en from '@/locales/en';
import Headline from '@/components/Text/HeadLine';

export default function BaseError() {
  return (
    <div className="mt-24">
      <Image
        priority
        src={exclamation as string}
        alt={en.exclamationSign}
        className="mx-auto mb-8"
      />
      <Headline classNames="mb-6 text-center">
        {en.sorrySmthWentWrong}
      </Headline>
    </div>
  );
}
