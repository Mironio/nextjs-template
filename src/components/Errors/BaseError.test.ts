import { expect, it } from 'vitest';
import BaseError from './BaseError';

it('should match snapshot', () => {
  expect(BaseError()).toMatchSnapshot();
});
