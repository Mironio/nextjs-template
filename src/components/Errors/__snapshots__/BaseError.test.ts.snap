// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
<div
  className="mt-24"
>
  <ForwardRef
    alt="Exclamation Sign"
    className="mx-auto mb-8"
    priority={true}
    src="/public/images/exclamation.svg"
  />
  <Headline
    classNames="mb-6 text-center"
  >
    Sorry something went wrong
  </Headline>
</div>
`;
